test/Sentry.Tests/HubTests.cs

```diff
@@ -1290,14 +1290,10 @@ public void CaptureTransaction_EventProcessor_Gets_ScopeAttachments()
         hint.Attachments.Should().Contain(attachments);
     }
 
-#if ANDROID && CI_BUILD
-    // TODO: Test is flaky in CI
-    [SkippableTheory(typeof(NSubstitute.Exceptions.ReceivedCallsException))]
-#else
     [Theory]
-#endif
     [InlineData(false)]
-    [InlineData(true)]
+    // [InlineData(true)]
+    [Repeat(20, true)]
```

Comment:
This test passes 20 times on each CI server (Repeat(20, true))... across 5 x different versions of Android that's 100 test passes. So far then, I've been unable to reproduce the original _flakiness_.

Initially I tried setting this to repeat 100 times but the [Android CI servers fell over when I did that](https://github.com/getsentry/sentry-dotnet/actions/runs/5585579820/jobs/10208679170)... Only the Android CI servers, so there is something special about them.